starcheck/src/starcheck.pl

```diff
@@ -550,26 +549,20 @@ sub json_obsids{
 close($JSON_OUT);
 
 # Produce final report
-my %save_hash;
```

Comment:
While I was in the report printing I removed an old idea that was never finished to save out parameters of the run.